src/Sylius/Bundle/ApiBundle/Serializer/Denormalizer/CommandDenormalizer.php

```diff
@@ -63,6 +63,11 @@ public function denormalize($data, $type, $format = null, array $context = [])
         }
     }
 
+    public function getSupportedTypes(?string $format): array
+    {
+        return ['object' => true];
```

Comment:
We should probably play around with caching in these, but that's a topic for a later date.

Reply:
For this denormalizer caching is enabled as you in this line

Reply:
Yeah, but the vast majority is set to `false` and I meant it in an overall sense.